### DISCUSSION TITLE
Analyze CPU Utilization (reports/database)

### DISCUSSION THREAD

**Discussion by wordbricks-dev** [supabase/supabase#15595] [Questions]
https://supabase.com/dashboard/project/[my-project]/reports/database shows Max CPU usage.

It hit 99.7% and Is there way to know what cause this CPU peak?

<img width="184" alt="image" src="https://github.com/supabase/supabase/assets/123422550/a40d5e68-9fd7-4a78-a1f7-bfc08c3e8d8b">

**Comment by GaryAustin1** [comment]
You might check the query perfomance report...
![image](https://github.com/supabase/supabase/assets/54564956/67ee1086-7028-4211-9a1a-a361fbff5bfb)
